bundles/org.openhab.binding.bticinosmarther/README.md

```diff
@@ -0,0 +1,328 @@
+
+* Flush mounting installation item (X8000)
+* Wall installation item (X8000W)
+
+If you can control them from BTicino Thermostat mobile app on your iPhone/Android you should be able to add it as a thing.
+
+## Discovery
+
+As long as BTicino Smarther Chronothermostat devices are available in the locations registered on the user account configured with the bridge they should show up whenever you initiate discovery of things.
+
+If no devices are showing up, try to connect to the device(s) from your smartphone to make sure the device(s) are in use by your user account.
+
+The discovery of devices in the Smarther API is based on what is known by Legrand.
+There is difference between e.g. smartphones and computers which can discover devices on the local network and the Smarther API which is not able to do so; it only knows about a device if your account is currently associated with the device.
+
+## Channels
+
+### Bridge
+
+The channels on the bridge are the ones used to get details of current communication with Smarther API on the Legrand account associated with the bridge.
+
+**Status Channels:**
+
+The following channels represent the current operational status of the bridge and must all be referenced with the `status#` prefix.
+
+| Channel Type ID   | Item Type            | Read/Write | Description                                                                            | Type     |
+|-------------------|----------------------|------------|----------------------------------------------------------------------------------------|----------|
+| apiCallsHandled   | Number               | Read-only  | The total number of API calls handled by the bridge                                    | Common   |
+| notifsReceived    | Number               | Read-only  | The total number of push notifications received by the bridge                          | Common   |
+| notifsRejected    | Number               | Read-only  | The total number of push notifications rejected by the bridge                          | Common   |
+
+**Configuration Channels:**
+
+The following channels represent convenience configuration channels for the bridge and must all be referenced with the `config#` prefix.
+
+| Channel Type ID   | Item Type            | Read/Write | Description                                                                            | Type     |
+|-------------------|----------------------|------------|----------------------------------------------------------------------------------------|----------|
+| fetchConfig       | Switch               | Read-write | Trigger to manually fetch updated bridge configuration from Smarther API               | Advanced |
+| accessToken       | String               | Read-only  | The current accessToken used in communication with Smarther API                        | Advanced |
+
+### Devices
+
+The temperature channels have dimension `Number:Temperature` and a precision of one tenth degree Celsius.
```

Comment:
This is given in the table below - can this sentence here thus be removed?

Reply:
Is there just to give the detail on managed precision. Is it better to add it to field description (in the table) and remove the row ?

Reply:
Yes, the info about dimension is duplicated here and the the one about precision is not given for other numbers in the table - so yes, I'd think it makes sense to add the info directly in the table for the respective channels.